packages/internals/support/src/validator.js

```diff
@@ -6,6 +6,11 @@ class Validator extends Ajv {
   constructor(...args) {
     super(...args);
     ajvKeywords(this, ['instanceof', 'typeof', 'regexp']);
+    const definition = require('ajv-keywords').get('instanceof').definition;
```

Comment:
Is it worth adding the collections here?

Reply:
Yes definitely, will add that in as a future separate PR though.